### PR TITLE
Getting shot in the head instantly kills you. Nanotrasen recommends wearing helmets.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request
Getting shot in the head instantly kills you.

## Why It's Good For The Game

Discord_OLjdMBxuS7

let's give it a try and see who's right

## Changelog
:cl:
balance: Getting shot in the head instantly kills you. Nanotrasen recommends wearing helmets.
/:cl:

### CODE REVIEW

**Comment by axietheaxolotl** [comment]
I do think this is a good idea 👍

**Comment by tralezab** [comment]
you die of brain damage? like... you suffer brain death?
??

**Comment by Iamgoofball** [comment]
@tralezab yeah the bullet fucks your brain up

**Comment by dawsonkeyes** [comment]
you made your last feature pr, kid
sorry you got twisted up in this scene
from where you’re kneeling it must look like a 2-key close of bad luck
but the truth is, the repo was rigged from the start

**Comment by necromanceranne** [comment]
[Song of the PR](https://www.youtube.com/watch?v=JskaByVlXqY)

**Comment by tralezab** [comment]
> @tralezab yeah the bullet fucks your brain up

i misunderstood, awesome pr

File: code/modules/projectiles/projectile.dm [hunk 1]
```diff
@@ -302,6 +302,10 @@
 		var/limb_hit = hit_limb
 		if(limb_hit)
 			organ_hit_text = " in \the [parse_zone(limb_hit)]"
+			if(limb_hit == BODY_ZONE_HEAD)
+				if(ishuman(L) && !nodamage && !blocked)
```
[the remark this answers is not in the document]
Comment (Iamgoofball) [reply code/modules/projectiles/projectile.dm:306]: thats what the blocked check is

**Comment by timothymtorres** [comment]
You need to test different guns:

- Syringe guns
- Beanbag rounds
- Xeno spit
- Wands
- Laser guns
- Foam darts
- Ebow guns
- Magic missle/Spells
- Shotgun blasts (can you kill several people at once with one shot since it spreads)

Anything that fires ANY kind of projectile that does even 1 brute damage now has the potential to be a insta death weapon.  Considering that nobody has access to helmets besides security, this is a really bad idea.

-1 from me unless you add a damage cap that is a high bar for entry (20 damage projectile)

**Comment by LemonInTheDark** [comment]
Testmerged on terry

**Comment by newfren** [comment]
this is a fucking terrible idea BUT it is extremely funny

**Comment by timothymtorres** [comment]
dreamseeker_mmN4wIsWMU

**Comment by Striders13** [comment]
this does include riot foam darts. Awesome

**Comment by Iamgoofball** [comment]
less than lethal != non-lethal, always remember that

File: code/modules/projectiles/projectile.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (necromanceranne) [reply code/modules/projectiles/projectile.dm:306]: Why the fuck are these arguments never consistent, is that seriously armor and not shield block?

[the remark this answers is not in the document]
Comment (Iamgoofball) [reply code/modules/projectiles/projectile.dm:306]: I think so?

**Comment by Serxule** [comment]
arent you early for an april fools pr?

**Comment by Timberpoes** [comment]
Ahhh yes, the golden BB

**Comment by WinterDarkraven** [comment]
I like the concept. Perhaps damage to limbs should also damage organs, too -- instead of it being an insta kill. I believe Bay has a system like this. Would be neat to see hollow points being useful to shred organs in a few shots.

**Comment by Serxule** [comment]
> I like the concept. Perhaps damage to limbs should also damage organs, too -- instead of it being an insta kill. I believe Bay has a system like this. Would be neat to see hollow points being useful to shred organs in a few shots.

adding baymed to tg is the easiest way to kill the server, also why would limbs affect organs?

**Comment by Helg2** [comment]
Turn on respawn, and it'll become even funnier.

**Comment by Serxule** [comment]
Screenshot_3000

**Comment by Serxule** [comment]
> Because organs are in limbs lol.
> 
> Hit the head, brain/eye damage. Hit chest, organ damage. Hit right arm, if they have a tools implant then it should get damaged.

that pretty much only applies to chest/head, "limbs" generally refers to arms and legs, which dont have organs in them (and rarely do people have implants)

**Comment by LukasBeedellCodestuff** [comment]
"Why do coders and players have a bad relationship" 

average pr:

If you're going to make this at least make it so only lethal bullets kill.

**Comment by HansRobertCoooding** [comment]
Most sane goofball PR

**Comment by Thunder12345** [comment]
> Testmerged on terry

The story so far:
In the beginning this PR was created.
This has made a lot of people very angry and been widely regarded as a bad move.

**Comment by zxaber** [comment]
There should *probably* be a damage threshold.

**Comment by Fikou** [comment]
comedy prs can tm'd while closed

**Comment by Overuseddart** [comment]
Wait so the sec grenades that fire shrapnel just insta kill now? And the best murderborne item is the riot foam dart pistol? Can we full merge this?

**Comment by ChristianDenDanske** [comment]
I can't see why this didn't get speed merged.

**Comment by GoodDays13** [comment]
space six siege

**Comment by actioninja** [comment]
> [Song of the PR](https://www.youtube.com/watch?v=JskaByVlXqY)

[alternatively](https://www.youtube.com/watch?v=ZmgNZHCoPWI)

**Comment by Kylerace** [comment]
> [Song of the PR](https://www.youtube.com/watch?v=JskaByVlXqY)
Screenshot_2372

**Pull request closed without merging**